index.html

```diff
@@ -11,123 +11,132 @@
 
     <!-- Navigation section -->
     <header>
-        <nav><img src="./images/ic_menu.svg" alt="Menu" title="Menu"></nav>
+        <nav class="nav">
+            <img class="nav__icon" src="./images/ic_menu.svg" alt="Menu" title="Menu">
+            <ul class="nav__links">
+                <li><a href="#" class="nav__link"><h2>Home</h2></a></li>
+                <li><a href="#" class="nav__link"><h2>Portfolio</h2></a></li>
+                <li><a href="#" class="nav__link"><h2>About</h2></a></li>
+                <li><a href="#" class="nav__link"><h2>Contact</h2></a></li>
+            </ul>
+        </nav>
     </header>
 
     <!-- Hero section -->
     <section class="hero-container">
-        <h1 class="title">Charlie Giorgania</h1>
-        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.<br>
+        <h1 class="hero__title hero__grid__item title">Charlie Giorgania</h1>
+        <p class="hero__text hero__grid__item paragraph">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.<br>
         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-        <button type="button" class="bg-dark">
+        <button type="button" class="hero__button hero__grid__item button btn-bg-dark">
             Start Collaboration
         </button>
-
-<!-- Projects section -->
     </section>
         
     <!-- Projects section -->
     <section class="projects-container">
-        <section class="projects-title-container">
-            <h2 class="title">Projects</h2>
-        </section>
         <section class="projects-cards-container">
-            <article class="project-card">
-                <img src="./images/firstProject.svg" alt="Project one" title="Project one">
+            <section class="projects-title-container">
+                <h2 class="projects__title title">Projects</h2>
+            </section>
+            <article class="project-card  bg-dark card1 bg-none">
+                <img class="project__image" src="./images/firstProject.svg" alt="Project one" title="Project one">
                 <div class="projects-text">
                     <h3>Project name goes here</h3>
                     <ul>
                         <li>HTML/CSS</li>
                         <li>Ruby on Rails</li>
                         <li>JavaScript</li>
                     </ul>
-                    <button class="projects-button bg-dark" type="button">
+                    <button class="button projects-button btn-bg-dark" type="button">
```

Comment:
The featured project has not been implemented in the desktop version. It will be nice to fix this.
project design:
![image](https://user-images.githubusercontent.com/51296741/160030049-c791369c-0ac4-45c3-b8ec-b692b0f89fe7.png)

Your design:
![image](https://user-images.githubusercontent.com/51296741/160030142-785e7576-1823-4e34-9d1c-2c01ee218c61.png)
And the no margin is to the right. And not to left. I will suggest you fix this also.

Reply:
@enaburekhan Isn't the project details a hover state of the image?